content/03.methods.md

```diff
@@ -116,11 +116,12 @@ We used Manta SV [@doi:10/gf3ggb] v1.4.0 for structural variant (SV) calls.
 Manta SV calling was also limited to regions used in Strelka2.
 The hg38 reference for SV calling used was limited to canonical chromosome regions.
 The somatic DNA workflow for SNV, INDEL, copy number, and SV calling can be found in the [KidsFirst Github repository](https://github.com/kids-first/kf-somatic-workflow).
+Manta SV output was annotated using [AnnotSV](https://lbgi.fr/AnnotSV/) [@doi:10.1093/bioinformatics/bty304] and the workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/annotSV/tree/master).
```

Comment:
Can we specify the version of AnnotSV that was used?

Reply:
@kgaonkar6 - can you alter the CWL to pull the version of AnnotSV used and also let us know which version here? Thank you!